main.go

```diff
@@ -69,20 +73,96 @@ func handler(w http.ResponseWriter, r *http.Request) {
 }
 
 func (s *extAuthzServerV3) Check(ctx context.Context, req *authv3.CheckRequest) (*authv3.CheckResponse, error) {
+	// Parse request
 	attrs := req.GetAttributes()
 
-	// Print each attribute individually
-	for key, value := range attrs.GetRequest().GetHttp().GetHeaders() {
-		fmt.Printf("Header: %s = %s\n", key, value)
+	// Load policies from files
+	policies, err := policy.Load(s.policies...)
+	if err != nil {
+		log.Printf("Failed to load policies: %v", err)
+		return nil, err
+	}
+
+	resource, err := request.Convert(attrs)
+	if err != nil {
+		log.Printf("Error converting request: %v", err)
+		return nil, err
 	}
-	// Print the entire struct with field names
-	fmt.Printf("Attributes: %+v\n", attrs)
 
-	// Implement your authorization logic here
-	// For now, allow all requests
+	engine := jsonengine.New()
+	var responses []jsonengine.Response
+	responses = append(responses, engine.Run(ctx, jsonengine.Request{
```

Comment:
won't we have just one response every time check runs? for each request?

Reply:
yes their should be one response , i have made changes